plugins/blip/basics/components/CalendarContainer.js

```diff
@@ -28,6 +28,9 @@ var BasicsUtils = require('./BasicsUtils');
 var ADay = require('./day/ADay');
```

Comment:
Hmm, that *should* be fixed as part of [this commit](https://github.com/tidepool-org/tideline/pull/299/commits/c18f60e6c0fd60d41310bb3816e5847e561c036e#diff-45ff9e6756044f644ca92e6381755b8f), but I'll be sure to double check.